udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,14 +55,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Query query = new Query("CommentSingle").addSort("timestamp", SortDirection.DESCENDING);
 
     PreparedQuery results = datastore.prepare(query);
-
+    int limiter = getCommentCount(request);
```

Comment:
I'd suggest naming this `commentLimit`, since `limiter` sounds like an object that does the limiting.

Reply:
Done